src/App.js

```diff
@@ -1,15 +1,33 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
-import PriSecBtn from "./components/PriSecBtn";
+import ClassesPage from "./components/pages/ClassesPage";
+import ClassSummaryPage from "./components/pages/ClassSummaryPage";
+import CompetitorsPage from "./components/pages/CompetitorsPage";
+import ContestsPage from "./components/pages/ContestsPage";
+import ExcersisesPage from "./components/pages/ExcersisesPage";
 import HomePage from "./components/pages/HomePage";
+import LoginPage from "./components/pages/LoginPage";
+import SingleSummaryPage from "./components/pages/SingleSummaryPage";
 
 function App() {
   return (
     <div className="App">
-      {/* #24/UI/Primary-secondary-btns - put here for test purposes only */}
-      <PriSecBtn primary text="PRIMARY" />
-      <PriSecBtn secondary text="SECONDARY" />
-      {/* ==== */}
-      <HomePage />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/login" element={<LoginPage />} />
+        <Route path="/contests" element={<ContestsPage />} />
+        <Route path="/contest/1/classes" element={<ClassesPage />} />
```

Comment:
These should receive the specific id in order to display the page for any contest or any class. Also in RR6 you can group them together, have a look at the documentation: https://reactrouter.com/docs/en/v6/getting-started/overview